### ISSUE TITLE
Suggestion: Use external audio to sync haptics

### ISSUE THREAD

**Issue by firstel** [metafetish/syncydink#68]
This would help VR Headset users by listening to the audio and comparing to the source video, then adjusting the current playback position to match.

For example there is no easy way to sync a video being played on an Oculus Go headset to devices which Synkydink can control without trial and error and lifting the headset on and off.

**Comment by qdot** [comment]
@firstel Just curious, how are you using Synycdink on an OGo currently? Playing the video manually while trying to hit play on a desktop/phone or something?

**Comment by firstel** [comment]
Playing the video using OGo app Pigasus Video Player from a local network server, and using Synkydink on a desktop Mac in a Chrome browser.

Sent from my mobile device. Please excuse any typos.

________________________________

**Comment by highfiiv** [comment]
Yes! @qdot I came here looking for a way to tap system audio to patch it into haptics like [web-based equalizers](https://codepen.io/WebDEasy/pen/PrmKbO) work.

There are many out there but it shouldnt be a long reach between system audio equalizer bands and haptic actions.
Can this be done?

**Comment by qdot** [comment]
For now, check out https://github.com/lucky-skebe/SoundSync/
